cmd/test/main.go

```diff
@@ -56,38 +67,56 @@ func NewTestCommand(ctx *TestCommandContext) *cobra.Command {
 }
 
 func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error {
-	s := spinner.New(spinner.CharSets[9], 100*time.Millisecond)
-
-	s.Suffix = " Loading..."
-	s.Color("cyan")
+	s := createSpinner(" Loading...", "cyan")
 	s.Start()
 
-	messageChannel := bl.PopulateVersionMessageChan(ctx.VersionMessageClient, ctx.CliVersion)
+	messages := make(chan *messager.VersionMessage, 1)
+	go ctx.Messager.LoadVersionMessages(messages, ctx.CliVersion)
 
-	config, err := ctx.LocalConfig.GetConfiguration()
+	validPaths, invalidPaths, _ := ctx.Validator.Validate(paths)
```

Comment:
kubeconform step is performed before the `createEvaluation` request. although it doesn't make much difference right now because they are unrelated steps, I feel like conseptually we should run the validation step after we create an evaluation.
In case there's a panic inside validate or whatever, this might break the flow without any record of it being run.
also, we will later add another step (yamlValidation) and I think we should keep the step 0 -> 1 -> 2 -> 3 structure